diff -Naur buildroot-2025.02.9-org/package/libnpupnp/libnpupnp.hash buildroot-2025.02.9/package/libnpupnp/libnpupnp.hash
--- buildroot-2025.02.9-org/package/libnpupnp/libnpupnp.hash	2025-12-11 09:20:34.000000000 +0100
+++ buildroot-2025.02.9/package/libnpupnp/libnpupnp.hash	2025-12-14 10:31:58.390661651 +0100
@@ -1,5 +1,5 @@
-# Hash from: http://www.lesbonscomptes.com/upmpdcli/downloads/libnpupnp-4.2.2.tar.gz.sha256
-sha256  cb3968773d30e2bfc765547df514fdc8927b9a37ecccca2e260ee8b612e756bc  libnpupnp-4.2.2.tar.gz
+# From https://www.lesbonscomptes.com/upmpdcli/downloads/libnpupnp-6.2.3.tar.gz.sha256
+sha256  563d2a9e4afe603717343dc4667c0b89c6a017008ac6b52262da17a1e4f6bb96  libnpupnp-6.2.3.tar.gz
 
 # Hash for license file:
 sha256  c8b99423cad48bb44e2cf52a496361404290865eac259a82da6d1e4331ececb3  COPYING
diff -Naur buildroot-2025.02.9-org/package/libnpupnp/libnpupnp.mk buildroot-2025.02.9/package/libnpupnp/libnpupnp.mk
--- buildroot-2025.02.9-org/package/libnpupnp/libnpupnp.mk	2025-12-11 09:20:34.000000000 +0100
+++ buildroot-2025.02.9/package/libnpupnp/libnpupnp.mk	2025-12-14 10:34:33.166230262 +0100
@@ -4,11 +4,11 @@
 #
 ################################################################################
 
-LIBNPUPNP_VERSION = 4.2.2
-LIBNPUPNP_SITE = http://www.lesbonscomptes.com/upmpdcli/downloads
+LIBNPUPNP_VERSION = 6.2.3
+LIBNPUPNP_SITE = https://www.lesbonscomptes.com/upmpdcli/downloads
 LIBNPUPNP_LICENSE = BSD-3-Clause
 LIBNPUPNP_LICENSE_FILES = COPYING
 LIBNPUPNP_INSTALL_STAGING = YES
 LIBNPUPNP_DEPENDENCIES = expat libcurl libmicrohttpd
 
-$(eval $(autotools-package))
+$(eval $(meson-package))
